dolphinscheduler-bom/pom.xml

```diff
@@ -536,7 +537,17 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-
+            <dependency>
+                <groupId>org.apache.kyuubi</groupId>
+                <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
+                <version>${kyuubi-jdbc.version}</version>
+                <exclusions>
```

Comment:
The `kyuubi-hive-jdbc-shaded` has NO transitive deps, please remove all exclusions

Reply:
OK